dev/tool/src/index.ts

```diff
@@ -1240,6 +1241,51 @@ export function devTool (
       }
     })
 
+  program
+    .command('restore-wiki-content-mongo')
+    .description('restore wiki document contents')
+    .option('-w, --workspace <workspace>', 'Selected workspace only', '')
+    .option('-d, --dryrun', 'Dry run', false)
+    .action(async (cmd: { workspace: string, dryrun: boolean }) => {
+      const params = {
+        dryRun: cmd.dryrun
+      }
+
+      let workspaces: Workspace[] = []
+      const accountUrl = getAccountDBUrl()
+      await withDatabase(accountUrl, async (db) => {
+        workspaces = await listWorkspacesPure(db)
+        workspaces = workspaces
```

Comment:
Please check only workspaces upgraded to current version only, not all.

Reply:
Done